tests/conf/karma.conf.js

```diff
@@ -0,0 +1,41 @@
+'use strict';
+
+module.exports = function (config) {
+    config.set({
+
+        basePath: '..',
+
+        frameworks: ['browserify', 'jasmine'],
+
+        browserify: {
+            debug: true,
+            transform: [
+                ['babelify', {
+                    presets: [
+                        'es2015',
+                    ],
+                }],
+            ],
+        },
+
+        files: [
+            '../node_modules/lodash/lodash.js',
+            '../node_modules/angular/angular.js',
+            '../node_modules/angular-cookies/angular-cookies.js',
+            '../node_modules/angular-mocks/angular-mocks.js',
```

Comment:
If we had loaded all the files with `import` then this wouldn't be necessary but as of now we're a bit stuck with this hybryd model